### PR TITLE
add caption, link to what is currently home page

### PR DESCRIPTION
**Pull request by paulvi**
the only description now is at http://marketplace.eclipse.org/content/bndtools-plugins-pelagic-stable

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,8 @@
-# Building
+# Bndtools Plugins by Pelagic
+
+see <http://marketplace.eclipse.org/content/bndtools-plugins-pelagic-stable>
```
Comment (fhuberts) [inline README.md]: See

Please also add link to https://marketplace.eclipse.org/content/bndtools-plugins-pelagic

**Comment by paulvi** [comment]
now with both links

**Comment by fhuberts** [comment]
Please try to stick to (generic) commit guidelines, see for example http://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html

Also sign-off your commits

File: README.md (hunk 1 above)
Comment (fhuberts) [inline README.md]: You did not change see into See

**Comment by fhuberts** [comment]
And you can squash the commits

**Comment by paulvi** [comment]
I do small changes to many open source project I am interested to understand if the author is active, welcoming commits, so I would not waste my time on work that would not be needed by authors

**Comment by paulvi** [comment]
Many projects even don't have good README or reference home page, i.e. just code

**Pull request merged**